### PR TITLE
#167958128 Create Article 

### PR DESCRIPTION
**Pull request by codexempire**
#### What does this PR do?
- This PR attends to the publishing and drafting of an article by an authenticated user

#### Description of Task to be completed?
- Enable publishing and drafting of articles

#### How should this be manually tested?
- run `npm install` to install dependencies
- run `npm run start:dev` to start the development server
- navigate to `localhost:5000/newArticle` in your browser

#### What are the relevant pivotal tracker stories?(if applicable)
167958128

#### Screenshots
[StoryID](url)

### CODE REVIEW

File: src/pages/Article/newArticle.spec.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+const body = {
+  blocks: [{ type: 'image', data: { file: { url: 'https://google.com' } } }]
+};
+const emptyBody = { blocks: [] };
+const {
+  file: { url }
+} = body.blocks[0].data;
+const defaulImage = getImage(emptyBody);
+
+test('getImage function returns the first image in the article', () => {
```
Comment (segunolalive) [inline src/pages/Article/newArticle.spec.js]: follow the test convention

File: src/pages/Article/newArticle.spec.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    store = createStore(true, false);
+    const { getByTestId } = renderComponent(store);
+    expect(getByTestId('title')).toBeDefined();
+    expect(getByTestId('description')).toBeDefined();
+    expect(getByTestId('editorjs')).toBeDefined();
+    expect(getByTestId('tags')).toBeDefined();
+    expect(getByTestId('publish-button')).toBeDefined();
+    expect(getByTestId('draft-button')).toBeDefined();
+  });
+  it('should redirect to the login page', () => {
```
Comment (segunolalive) [inline src/pages/Article/newArticle.spec.js]: this test block does not prove that there was a redirect to login page. Also, your test description should capture when/why

**Review by segunolalive — COMMENTED** [review]
Good work. Finish up.

File: src/components/SocialMedia/SocialMediaButtons.js [hunk 3]
```diff
@@ -4,9 +4,9 @@ import Google from './icons/google_btn.svg';
 import Github from './icons/login_github.svg';
 import './social.css';
 
-const GITHUB_LOGIN = `http://localhost:3000/api/v1/auth/github`;
-const FACEBOOK_LOGIN = `${process.env.API_BASE_URL}/auth/facebook`;
-const GOOGLE_LOGIN = `https://errorswag-staging.herokuapp.com/api/v1/auth/google`;
+const GITHUB_LOGIN = `https://errorswag-staging.herokuapp.com/api/v1/auth/github`;
+const FACEBOOK_LOGIN = `https://errorswag-staging.herokuapp.com/api/v1/auth/facebook`;
+const GOOGLE_LOGIN = `http://localhost:3000/api/v1/auth/google`;
```
Comment (tohbay) [inline src/components/SocialMedia/SocialMediaButtons.js]: This `GOOGLE_LOGIN should connect to the hosted api link

**Review by tohbay — APPROVED** [review]
Good job 👍 
Noticed you reverted the GOOGLE social Login link

**Review by kechyy — CHANGES_REQUESTED** [review]
Good Job @Princewill. However, I noticed there are some vital feedback given already, please attend to them.

**Review by kechyy — APPROVED** [review]

**Review by Onyimatics — APPROVED** [review]
Looks good to me. Welldone guys!

**Pull request merged**